experimental/dds/tree2/src/feature-libraries/default-field-kinds/defaultChangeFamily.ts

```diff
@@ -171,22 +173,86 @@ export class DefaultEditBuilder implements ChangeFamilyEditor, IDefaultEditBuild
 		destIndex: number,
 	): void {
 		const moveId = this.modularBuilder.generateId(count);
```

Comment:
What do you think about implementing this by rebasing one of the move fragments over the other?

Reply:
I thought about it but it seems like it would violate two assumptions we've been making in the past:
1. That both end points of moves are always present
2. That we don't rebase a change over and change with the same revision